models/articles-model.js

```diff
@@ -17,19 +17,38 @@ function selectArticleById(article_id) {
    })
 }
 
-function selectArticles(articles, comments){
+function selectArticles(articles, comments, topic){
+    const queryValues = []
+    let sqlQuery = 'SELECT * FROM articles';
+
+    if (topic){
+        queryValues.push(topic)
+        sqlQuery += `COUNT(comments.article_id) AS comment_count
+             FROM articles
+             LEFT JOIN comments
+             ON articles.article_id = comments.article_id 
+             WHERE TOPIC = $1
+             GROUP BY
+             articles.article_id
+             ORDER BY articles.created_at DESC;`,
+             [topic]
+        return queryValues
+    }
```

Comment:
the `return queryValues` on line 35 would be returning an array rather than a database query, so I would imagine this isn't working how you want it to in the current state